src/app/join/page.tsx

```diff
@@ -3,17 +3,25 @@
 
 import { useForm } from "react-hook-form";
 import { yupResolver } from "@hookform/resolvers/yup";
+import { schema } from "../schemas/participants";
 import { ParticipantDetails } from "@/components/participantDetails";
 import type { IParticipantDetails } from "../schemas/participants";
-import { schema } from "../schemas/participants";
-import { toast } from "sonner";
-import { useRef, useState } from "react";
+// import { useRouter } from 'next/navigation';
+
+import { useEffect, useRef, useState } from "react";
 import ky, { type HTTPError } from "ky";
+import { useSearchParams } from 'next/navigation'
+
+import { toast } from "sonner";
 import Dialog, { CopyElement } from "@/components/dialog";
 
 export default function Join() {
   const dialog = useRef(null);
   const [edit, setEdit] = useState("");
+
+  const searchParams = useSearchParams()
+  const editCode = searchParams.get('event-code')

```

Comment:
why `editCode`? could we make this `eventCode` for better cide readability

Reply:
oh right, didnt see that. also let's stick to snake_case for variable names, I find that more readable than camelCase